src/Microsoft.PowerShell.Commands.Management/commands/management/PingPathCommand.cs

```diff
@@ -156,49 +162,60 @@ protected override void ProcessRecord()
         {
             CmdletProviderContext currentContext = CmdletProviderContext;
 
-            foreach (string path in _paths)
+            if (_paths != null && _paths.Length != 0)
```

Comment:
We usually use short code path:
```suggestion
            if (_paths == null || _paths.Length == 0)
            {
                return false;
            }
```